### ISSUE TITLE
Monthly round stats show some results as "Gang Takover Failed" on non-gang rounds

### ISSUE THREAD

**Issue by YakumoChen** [nfreader/newSS13tools#58]
I was trying to find to see if there were any stats for nuke ops humiliating defeat but had an undefined and "gang loss" stats instead, which were confusing. Other modes had it too.

**Comment by nfreader** [comment]
Yeah, August stats are all sorts of screwy because of a change to how they were recorded. I'll see if I can fix that.
